lib/config/fileconf.go

```diff
@@ -1167,6 +1170,8 @@ type Proxy struct {
 	WebAddr string `yaml:"web_listen_addr,omitempty"`
 	// TunAddr is a reverse tunnel address
 	TunAddr string `yaml:"tunnel_listen_addr,omitempty"`
+	// PeerAddr is the address this proxy will be dialed at by its peers.
+	PeerAddr string `yaml:"peer_listen_addr,omitempty"`
```

Comment:
I think we should split this into a `peer_listen_addr` and `peer_public_addr` like other listeners; the address we're listening on is not necessarily the routable address that other proxies should use.

Reply:
I want to think about it a little more as it seems like it would require some changes to the proxy peering clients and ServerSpec. I don't think this needs to be in the initial implementation so I will create an issue to follow up on this.